docs/changelog.md

```diff
@@ -5,6 +5,345 @@ description: |
+  - Fixed compilation output of a single weak page break
+
+- Text
+  - Tuned hyphenation: It is less eager by default and hyphenations close to the
+    edges of words are now discouraged more strongly
+    (**May lead to larger layout reflows**)
+  - Setting the font to an unavailable family will now result in a warning
+  - Implemented a new smart quote algorithm, fixing various bugs where smart
+    quotes weren't all that smart
+  - Added [`text.costs`] parameter for tweaking various parameters that affect
+    the choices of the layout engine during text layout
+  - Added `typm` highlighting mode for math in [raw blocks]($raw.lang)
+  - Added basic i18n for Galician, Catalan, Latin, Icelandic, Hebrew
+  - Implemented hyphenation duplication for Czech, Croatian, Lower Sorbian,
+    Polish, Portuguese, Slovak, and Spanish.
+  - The [`smallcaps`] function is now an element function and can thereby be
+    used in show(-set) rules.
+  - The [`raw.theme`] parameter can now be set to `{none}` to disable
+    highlighting even in the presence of a language tag, and to `{auto}` to
+    reset to the default
+  - Multiple [stylistic sets]($text.stylistic-set) can now be enabled at once
+  - Fixed the Chinese translation for "Equation"
+  - Fixed that hyphenation could occur outside of words
+  - Fixed incorrect layout of bidirectional text in edge cases
+  - Fixed layout of paragraphs with explicit trailing whitespace
+  - Fixed bugs related to empty paragraphs created via `#""`
+  - Fixed accidental trailing spaces for line breaks immediately preceding an
+    inline equation
+  - Fixed [`text.historical-ligatures`] not working correctly
+  - Fixed accidental repetition of Thai characters around line breaks in some
+    circumstances
+  - New font metadata exceptions for Archivo, Kaiti SC, and Kaiti TC
+
+- Math
+  - Block-level equations can now break over multiple pages. This behaviour can
+    be disabled via `{show math.equation: set block(breakable: false)}`
+  - Matrix and vector sizing is now more consistent across different cell
+    contents
+  - Improved layout of attachments on parenthesized as well as under- or
+    overlined expressions
+  - Improved layout of nested attachments resulting from code like
+    `[#let a0 = $a_0$; $a0^1$]`
+  - Improved layout of primes close to superscripts
+  - Typst now makes use of math-specific height-dependant kerning information in
+    some fonts for better attachment layout
+  - The `floor` and `ceil` functions in math are now callable symbols instead,
+    such that `[$ floor(x) = lr(floor.l x floor.r) $]`
+  - The [`mat.delim`]($math.mat.delim), [`vec.delim`]($math.vec.delim), and
+  - [`cases.delim`]($math.cases.delim) parameters now allow any character that
+    that is considered as a delimiter or "fence" (e.g. |) by Unicode. The
+    `{delim: "||"}` notation is _not_ supported anymore and should be replaced
+    by `delim: bar.double` (**Minor breaking change**)
```

Comment:
But it's still full of typos..?
```suggestion
    [`cases.delim`]($math.cases.delim) parameters now allow any character that
    is considered as a delimiter or "fence" (e.g. |) by Unicode. The
    `{delim: "||"}` notation is _not_ supported anymore and should be replaced
    by `{delim: bar.double}` (**Minor breaking change**)
```

Reply:
well, full of typos is a bit of an exaggaration. There is one typo (double "that") there, which I missed because of the multi-line diff. (I couldn't copy your suggestion because it actually also contained a new typo: `"is _no longer_ support"`.) The curly braces don't really do anything here.